config/initializers/grover.rb

```diff
@@ -1,6 +1,16 @@
+# https://github.com/Studiosity/grover/blob/main/README.md#debugging
+# The headless option disabled is not compatible with exporting of the PDF.
+#
 Grover.configure do |config|
   config.options = {
     executable_path: '/usr/bin/chromium-browser',
     launch_args: ['--disable-dev-shm-usage'],
+    format: 'A4',
+    margin: {
+      top: '20px',
```

Comment:
@instantrick we can also define a margins here in either `px` or `cm`

Reply:
@peterdavidhamilton please can you change the top to 5px and the bottom to 5px so. can see if it makes any difference as I suspect that there are other factors overriding these settings.